lib/result-view.coffee

```diff
@@ -130,14 +130,15 @@ class ResultView
                 @element.classList.add 'rich'
                 @setMultiline true
 
+            container.appendChild htmlElement
+
             if mimeType is 'text/html'
-                container.classList.add('html')
+                if @getAllText() isnt ''
+                    @element.classList.remove 'rich'
```

Comment:
Could you add a comment [here](https://github.com/lgeiger/hydrogen/blob/issue-26/lib/result-view.coffee#L136) to remind our future selves that `@getAllText()` must be called after appending the htmlElement?